src/azul/plugins/metadata/hca/transform.py

```diff
@@ -986,6 +980,22 @@ def _samples(self, samples: Iterable[api.Biomaterial]) -> MutableJSON:
                 assert False, sample
         return result
 
+    def _is_dcp_matrix(self, file: api.File) -> bool:
+        submitter = Submitter.for_file(file)
```

Comment:
can `submitter.category_for_file()` be used here? 
Thinking about ways to de-duplicate.